custom-metrics-stackdriver-adapter/examples/prometheus-to-sd/custom-metrics-prometheus-sd.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    run: custom-metric-prometheus-sd
+  name: custom-metric-prometheus-sd
+  namespace: default
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      run: custom-metric-prometheus-sd
+  template:
+    metadata:
+      labels:
+        run: custom-metric-prometheus-sd
+    spec:
+      containers:
+      - command:
+        - /bin/sh
+        - -c
+        - ./prometheus_dummy_exporter --metric-name=foo --metric-value=40 --port=8080
+        env:
+        - name: POD_ID
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: metadata.uid
+        image: gcr.io/bskiba-dev/prometheus-dummy-exporter:v0.1.0 
+        name: prometheus-dummy-exporter
+        resources:
+          requests:
+            cpu: 100m
+      - name: prometheus-to-sd
+        image: gcr.io/bskiba-dev/prometheus-to-sd:v0.2.1
```

Comment:
Please use official release of prom-to-sd.

Reply:
Done. Note that I will have to use version 0.2.2 once it is available, as this is blocked on merging and releasing #75